strr-base-web/app/utils/permitDashHelpers.ts

```diff
@@ -120,10 +120,15 @@ export const getDashboardAddresses = (business: StrrBusiness) => {
   return addresses
 }
 
+export function getPhoneDisplay (phone: ConnectPhone) {
```

Comment:
I have similar format helpers in examiners app, maybe worth moving to the base and consolidating...